### ISSUE TITLE
SerialUSB cause memory leak after LowPower.sleep(), using ArduinoLowPower lib. samd21 cpu

### ISSUE THREAD

**Issue by Terryp44** [arduino/ArduinoCore-samd#293]
Every wakeup causes some memory loss, eventually causing CPU reset. However if USB is not plugged in, no memory is lost.
using USBDevice.detach(); and USBDevice.attach();, open/close port does not help.
Using samd version 1.6.17
Test RAM memory usage via:
extern "C" char *sbrk(int i);
int FreeRam (void)
{
  char stack_dummy = 0;
  return &stack_dummy - sbrk(0);
}

**Comment by Terryp44** [comment]
After reconnecting the SerialUSB port to the PC. samd CPU ram memory decreases by 592 bytes every sleep cycle.

**Comment by Terryp44** [comment]
After reconnecting the SerialUSB port to the PC. samd CPU ram memory decreases by 592 bytes every sleep cycle.

**Comment by facchinm** [comment]
Hi @Terryp44 ,
I pushed #295 to solve the issue. Can you test the produced json and confirm that the standby "dance" doesn't leak anymore? Thanks!
